test/fixtures/flow/declare-module/8/options.json

```diff
@@ -0,0 +1,3 @@
+{
+  "throws": "Unexpected token (2:2)"
```

Comment:
Yeah something like that. Could even give an example (we haven't really done that)

I've been liking elm's errors/messages http://elm-lang.org/blog/compilers-as-assistants